src/components/Modals/SignIn.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import ModalCloseImg from '../../assets/icon/black-close-button.png';
+import * as S from './style';
+
+interface Props{
+}
+
+const SignIn: React.SFC<Props> =()=> {
```

Comment:
```suggestion
const SignIn: React.FC<Props> =()=> {
```
react SFC는 개발 중지됐습니다. FC로 사용하세요

Reply:
넵

Reply:
Stateless로 사용하고싶다면 StatelessComponent도 있습니다.